### DISCUSSION TITLE
How can I open app TokenPocket from another app on mobile

### DISCUSSION THREAD

**Discussion by PhamHang1491** [TP-Lab/Mobile-SDK#33] [Q&A] [answered]
In my game, I have a button ConnectTokenPocket. When clicking this button I want to launch the app TokenPocket and then open dapp link such as https://google.com
I tried with some deeplink below but it did not work, even It cannot launch app TokenPocket
**tpwallet://open/dapp/google.com
tpwallet://browser/dapp/google.com
tokenpocket://app/google.com
https://www.tokenpocket.pro/dapp/google.com**

If anyone knows this issue, please help me. Thanks so much.

**Comment by chendatony31** [comment, marked as answer]
This doc may help: https://help.tokenpocket.pro/developer-en/wallet/pull-up-wallet-with-deeplink

**Reply by PhamHang1491** [reply]
I got the solution. 
Thank you so much!

**Reply by PhamHang1491** [reply]
> This doc may help: https://help.tokenpocket.pro/developer-en/wallet/pull-up-wallet-with-deeplink

Hi @chendatony31,
When I use deeplink: _tpdapp://open?params={"url":"google.com"}_, it work on device Android only, for iOS, it's not.
Is there any different deeplink for iOS devices?

**Reply by chendatony31** [reply]
Hi, please try to update your iOS TokenPocket wallet.

**Reply by PhamHang1491** [reply]
I'm using v2.0.4. It was just released 1 day ago

**Reply by chendatony31** [reply]
It should works, you can input `tpdapp://open?params={"url":"google.com"}` in your safari to pull up TP wallet for test
